docs/cfn-template.yaml

```diff
@@ -0,0 +1,68 @@
+AWSTemplateFormatVersion: 2010-09-09
+Resources:
```

Comment:
Can you add a rule for AWS Health Events? It should look something like this:

```
  ScheduledChangeRule:
    Type: 'AWS::Events::Rule'
    Properties:
      EventPattern:
        source:
          - aws.health
        detail-type:
          - AWS Health Event
      Targets:
        - Id: 1
          Arn: !GetAtt Queue.Arn
```

Reply:
Rule has been added